.github/workflows/ci-integration-tests-new.yml

```diff
@@ -141,67 +138,40 @@ jobs:
         run: |
-          cd $GITHUB_WORKSPACE/test/catalog/with_qvci
-          ./run.sh
+          # cd $GITHUB_WORKSPACE/test/catalog/with_qvci
+          # ./run.sh
           cd $GITHUB_WORKSPACE/test/catalog/without_qvci
           ./run.sh
-      
+
       - name: Run nopi tests
         run: |
-          sed -i "s/hostname=dbt.*/hostname='$DBT_TERADATA_SERVER_NAME_2'/g" tests/conftest.py
-          sed -i "s/username=demo.*/username='$DBT_TERADATA_USERNAME_2'/g" tests/conftest.py
-          sed -i "s/password=dbt.*/password='$DBT_TERADATA_PASSWORD_2'/g" tests/conftest.py
+          sed -i "s/hostname=dbt.*/hostname='$DBT_TERADATA_SERVER_NAME'/g" tests/conftest.py
+          sed -i "s/username=demo.*/username='$DBT_TERADATA_USERNAME'/g" tests/conftest.py
+          sed -i "s/password=dbt.*/password='$DBT_TERADATA_PASSWORD'/g" tests/conftest.py
           pytest tests/functional/adapter/test_nopi.py
         env:
-          DBT_TERADATA_SERVER_NAME_2: ${{ vars.DBT_TERADATA_SERVER_NAME_2 }}
-          DBT_TERADATA_USERNAME_2: ${{ vars.DBT_TERADATA_USERNAME_2 }}
-          DBT_TERADATA_PASSWORD_2: ${{ vars.DBT_TERADATA_PASSWORD_2 }}
-
-      # - name: Stop Clearscape DBS
-      #   if: always()
-      #   shell: bash
-      #   run: |
-      #     curl -X 'PATCH' \
-      #       'https://api.clearscape.teradata.com/environments/dbt-teradata-1' \
-      #       -H 'accept: application/json' \
-      #       -H 'Authorization: Bearer 2OKQYw4jhaCO3T7UBLjYmsoTxs73IG9q4Ps3kZT6DpnKBKO2mLYd4k6QkGuwW9db' \
-      #       -H 'Content-Type: application/json' \
-      #       -d '{
-      #       "operation": "stop"
-      #     }'
-
-      #     curl -X 'PATCH' \
-      #       'https://api.clearscape.teradata.com/environments/dbt-teradata-2' \
-      #       -H 'accept: application/json' \
-      #       -H 'Authorization: Bearer 2OKQYw4jhaCO3T7UBLjYmsoTxs73IG9q4Ps3kZT6DpnKBKO2mLYd4k6QkGuwW9db' \
-      #       -H 'Content-Type: application/json' \
-      #       -d '{
-      #       "operation": "stop"
-      #     }'
-
-      #     curl -X 'PATCH' \
-      #       'https://api.clearscape.teradata.com/environments/dbt-teradata-3' \
-      #       -H 'accept: application/json' \
-      #       -H 'Authorization: Bearer 2OKQYw4jhaCO3T7UBLjYmsoTxs73IG9q4Ps3kZT6DpnKBKO2mLYd4k6QkGuwW9db' \
-      #       -H 'Content-Type: application/json' \
-      #       -d '{
-      #       "operation": "stop"
-      #     }'
-          
-      - uses: actions/upload-artifact@v2
-        if: ${{ failure() ||  cancelled() }}
-        with:
-          name: virtualbox-logs
-          path: /Users/runner/VirtualBox*
+          DBT_TERADATA_SERVER_NAME: ${{ steps.create-csae-environments.outputs.teradata-server-name }}
+          DBT_TERADATA_USERNAME: dbc
+          DBT_TERADATA_PASSWORD: ${{ secrets.CSAE_ENV_PASSWORD }}
+
+      - name: Delete CSAE environment
+        if: always()
+        run: ./.github/workflows/scripts/deleteTestEnvironments.sh
```

Comment:
Got below error

```
/home/runner/work/_temp/1171126b-a998-41a4-b1c4-bdf0d932675e.sh: line 1: 
./.github/workflows/scripts/deleteTestEnvironments.sh: Permission denied
Error: Process completed with exit code 126.
```

I think we need to give execute permission to this shell script